deployment/README.md

```diff
@@ -74,142 +76,159 @@ git checkout 2.0.10
+
+### 3.2 Installing
 
-### 3.1 - Copying credentials
-If the installation is done on different deployment machine then the credential generation, proceed with steps 1.1 and 1.2 (cloning the repo and building the docker image on the deployment machine. Copy the folder `deployment/credentials.prepared` from the secure machine to the deployment machine into 
-`<git-repo-root>/deployment` folder.
- 
-### 3.2 - Installing
 Run the installation from the `deployment` folder:
+
 ``` bash
 ./install-dockers.sh mainnet
 ```
 
-This will install all services using several docker compose files. On the first run it will configure block chain nodes and database instances according to the credentials and configurations.
+This will install all services using several `docker-compose` files. On the first run it will configure block chain nodes and database instances according to the credentials and configurations.
+
+## Updating Credentials and Configurations
+
+Once Attestation Suite is installed, you can change credentials as follows:
 
-## Updating credentials and configurations
+- Stop the containers:
 
-Once Attestation Suite is installed one can change credentials as follows:
-- stop the containers
 ``` bash
 ./stop-all.sh mainnet
 ```
-- carry out steps 1.4 and 1.5 on a secure machine nad step 2.1 (copy `credentials.prepare` folder to the deployment machine).
-- start all services
+
+- Carry out steps 1.4 and 1.5 on a secure machine and step 2.1 (copy `credentials.prepare` folder to the deployment machine).
+
+- Start all services:
+
 ``` bash
 ./start-all.sh mainnet
 ```
 
-Updating can be done on specific containers only. In this case only specific containers are stopped, steps 1.4, 1.5 are carried out on updated configs, but only changed secure credential configs are copied to the deployment machine into the folders mounted to the specific containers. Those containers are then restarted. Stopping and starting is carried out using `docker-compose` and handy scripts in `deployment` folder.
+Updating can be done on specific containers only. In this case only specific containers are stopped, steps 1.4 and 1.5 are carried out on updated configs, but only changed secure credential configs are copied to the deployment machine into the folders mounted to the specific containers. Those containers are then restarted. Stopping and starting is carried out using `docker-compose` and handy scripts in `deployment` folder.
+
+## Monitoring
 
-## Step 4. Monitoring
-Attestation Suite monitoring installation is documented [here](./../docs/monitor/monitor.md).
+See [Attestation Suite monitoring](./../docs/monitor/monitor.md).
 
-## Indexer syncing times
+## Indexer Syncing Times
 
 For 2 days:
```

Comment:
I sent a question to David Pangerl in the doc channel: https://flarenetworks.slack.com/archives/C02NURDPAQZ/p1684765082339239

Reply:
Here's what David says:

About indexer syncing times in general: "indexing in our case is collecting all blocks (and block transaction) from the networks for a specific time range in the past (in our case 2 days)"
About the Algorand running sync: we do this when we cannot index in the past (since the network does not have an indexer), so if we need data for 2 days we need to run it for 2 days.